sklearn/neighbors/base.py

```diff
@@ -101,13 +101,14 @@ def _get_weights(dist, weights):
 class NeighborsBase(six.with_metaclass(ABCMeta, BaseEstimator)):
     """Base class for nearest neighbors estimators."""
 
-    @abstractmethod
-    def __init__(self):
-        pass
+    _fit_X = None
```

Comment:
Actually, it's not that unknown as I said. test fails on AttributeError if I omit them. but can be resolved easily. Will push the change soon